app/controllers/sign_up/registrations_controller.rb

```diff
@@ -29,7 +29,7 @@ def create
       if result.success?
         process_successful_creation
       else
-        render :new
+        render :new, formats: :html, locals: threatmetrix_variables
```

Comment:
Do we need `formats:` ? I know it's also in `#new`, but wondering if we need it there either.